rules/codelingo/effective-go/comment-first-word-as-subject/codelingo.yaml

```diff
@@ -295,12 +295,13 @@ tenets:
           The first sentence should be a summary that starts with the name being declared.
 
           ## Bad code
-          ```
+          ```golang
           // This func comment should begin with 'Bar'
           func Bar() {}
```

Comment:
If we aim to keep the only difference between the bad/good code to be the necessary change, hopefully then people can visually "diff" it.
```golang
// Return an empty string
func Foo() {
   return "";
}
```